python/lsst/meas/algorithms/pcaPsfDeterminer.py

```diff
@@ -369,7 +393,8 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
                         badCandidates.append(cand)
 
             badCandidates.sort(key=lambda x: x.getChi2(), reverse=True)
-            numBad = int(len(badCandidates) * (iter + 1) / self.config.nIterForPsf + 0.5)
+            numBad = numCandidatesToReject(len(badCandidates), iter,
```

Comment:
Can we change the variable name `iter` so it doesn't clash with a python function name? It's a bit confusing at first glance. I thought for a moment that you were passing the function reference into the routine.